helm/stargate/templates/stargate_coordinator_deployment.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stargate-coordinator
+  labels:
+    app: coordinator
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: coordinator
+  template:
+    metadata:
+      labels:
+        app: coordinator
+    spec:
+      containers:
+      - name: coordinator
+        {{ if eq .Values.cassandra.clusterVersion "3.11" }}
+        image: "{{ .Values.image.repository311 }}:{{ .Values.image.tag }}" 
+        {{ else if eq .Values.cassandra.clusterVersion "6.8" }}
+        image: "{{ .Values.image.repositoryDse68 }}:{{ .Values.image.tag }}"
+        {{ else }}
+        image: "{{ .Values.image.repository40 }}:{{ .Values.image.tag }}"
+        {{end}}
+        ports:
+        - containerPort: 8091
+        - containerPort: 9042
+        - containerPort: 8081
+        args:          
+         {{  range .Values.arguments }}
+           - {{ . }}
+         {{ end }}
+        readinessProbe:
+          httpGet:
+            path: /checker/readiness
+            port: 8084
+          timeoutSeconds: 20
+          initialDelaySeconds: 80
```

Comment:
initial delay of 80 is too long, we need something shorter, like:

 * initial delay 20s
 * delay between checks 5s
 * max checks 20
 
this means you will wait for a maximum of 1.5 mins for the service to be up..

This is also the reason why the API services were restarting, you mark this one as healthly earliest 80s after start, meaning others need to wait at least that for a successful bridge data call..